core-primitives/enclave-api/ffi/src/lib.rs

```diff
@@ -37,6 +37,11 @@ extern "C" {
 		latest_header_size: usize,
 	) -> sgx_status_t;
 
+	pub fn trigger_parentchain_block_import(
+		eid: sgx_enclave_id_t,
+		retval: *mut sgx_status_t,
+	) -> sgx_status_t;
```

Comment:
New e-call to specifically trigger the parentchain block import when we initialize the light-client. This is done to ensure we have a reasonably up-to-date view of the parentchain and that a potentially mega-large import does not have to be done when we're already in sidechain block production mode.

Reply:
Yes, just so you don't forget, the mega import will probably not take place if there is already other workers registered because we want to exchange the chain_db too when we onboard: https://github.com/integritee-network/worker/issues/273

Reply:
Indeed, the on-boarding might change the preconditions on this import trigger 👍 But it will still take a while until we have that.. so in the meantime we certainly have to trigger this import here.